LoopKit/InsulinKit/ExponentialInsulinModelPreset.swift

```diff
@@ -0,0 +1,87 @@
+//
```

Comment:
This file was previously located in Loop. It's the same except for the Afrezza model being added.

Reply:
Afrezza model was removed